src/components/bottomSheet/types.d.ts

```diff
@@ -48,7 +48,7 @@ export interface BottomSheetProps
    * snapPoints={['%100']}
    * @type Array<string | number>
    */
-  snapPoints: Array<string | number> | SharedValue<Array<string | number>>;
+  snapPoints?: Array<string | number> | SharedValue<Array<string | number>>;
```

Comment:
Is there anything we can do with the type annotation to tell users that this is only not required when the enableDynamicSizing prop is true?

Reply:
mmm not sure how could we achieve that, but if customers did not provide snap points and dynamic sizing prop, it will throw an exception https://github.com/gorhom/react-native-bottom-sheet/pull/1513/files#diff-620b7ea703632ea948b10134aaea17e0efd09781b64f587ecab4395812f9a3ccR26

Reply:
Just in the JSDoc comment above I was thinking. Should show in most IDEs when you hover over the prop or while you're typing

Reply:
@Eli-Nathan i have added an extra description in the latest commit